### PR TITLE
F move components out core folder

### PR DESCRIPTION
**Pull request by ahmad-demos**
- move components from /core/ folder to root, put them in their own folders.
- adjust importing.

### CODE REVIEW

**Review by spiess-demos — CHANGES_REQUESTED** [review]
Nice! A few nitpicks, and one concern: there are 2 tests failing because of the new structure.

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ Since v0.0.10, this Changelog is formatted according to the [Common Changelog][c
 
 ### Changed
 
+- ([#651](https://github.com/demos-europe/demosplan-ui/pull/651)) Move Components from /Core/ folder to root ([@ahmad-demos](https://github.com/@ahmad-demos))
```
[the remark this answers is not in the document]
Comment (ahmad-demos) [reply CHANGELOG.md]: https://github.com/demos-europe/demosplan-ui/pull/651/commits/5810a05d1ec88808e38c81b19fc2bb8b00acb461

File: src/components/DpEditor/DpLinkModal.vue [hunk 2]
```diff
@@ -46,12 +46,12 @@
...
-import DpButtonRow from '../../DpButtonRow/DpButtonRow'
-import DpCheckbox from '../../DpCheckbox/DpCheckbox'
-import DpInput from '../../DpInput/DpInput'
-import DpModal from '../../DpModal/DpModal'
-import { de } from '../../shared/translations'
-import { dpValidateMixin } from '../../../mixins'
+import DpButtonRow from '../DpButtonRow/DpButtonRow'
+import DpCheckbox from '../DpCheckbox/DpCheckbox'
+import DpInput from '../DpInput/DpInput'
+import DpModal from '../DpModal/DpModal'
```
[the remark this answers is not in the document]
Comment (ahmad-demos) [reply src/components/DpEditor/DpLinkModal.vue]: https://github.com/demos-europe/demosplan-ui/pull/651/commits/37fae6ef25dda48cd295781930437ab22b65b532

File: src/components/DpUpload/DpUpload.vue [hunk 3]
```diff
@@ -4,8 +4,8 @@
```
[the remark this answers is not in the document]
Comment (ahmad-demos) [reply src/components/DpUpload/DpUpload.vue]: https://github.com/demos-europe/demosplan-ui/pull/651/commits/eb890f81c357b8a4c5ff63e928fae9aa0987da9a

**Review by spiess-demos — CHANGES_REQUESTED** [review]
there are 2 test errors:

> Could not locate module ~/components/core/DpEditor/libs/handleWordPaste 
> Could not locate module ~/components/core/DpObscure.vue

**Review by spiess-demos — CHANGES_REQUESTED** [review]
there are still 2 test errors:

> Could not locate module ~/components/core/DpEditor/libs/handleWordPaste
> Could not locate module ~/components/core/DpObscure.vue

Please adapt paths here, as well.

**Comment by ahmad-demos** [comment]
> there are still 2 test errors:
> 
> > Could not locate module ~/components/core/DpEditor/libs/handleWordPaste
> > Could not locate module ~/components/core/DpObscure.vue
> 
> Please adapt paths here, as well.

Done in https://github.com/demos-europe/demosplan-ui/pull/651/commits/b9f7544e39c4e0c6749d5a103a8efea94e61c307

**Review by spiess-demos — APPROVED** [review]
lgtm & works - thanks!

**Pull request merged**